### ISSUE TITLE
No permissions and do not have a method or closure named "getFieldSettings"

### ISSUE THREAD

**Issue by rdomingue** [engram-design/Comments#26]
I've installed the plugin. I don't have the permissions tab like the screenshots are showing.
When I've try the comments form in my entry, I've got this error on submit : Craft\CommentsService and its behaviors do not have a method or closure named "getFieldSettings".

There is something I did wrong ?

**Comment by engram-design** [comment]
@rdomingue Sorry for the delay - This is fixed in the latest master, which will be incorporated in the next release. Permissions are almost finalised as well.

**Comment by rdomingue** [comment]
Ohhh nice I'm happy to hear it.
I was a bit confused when I installed it and didn't get the same as the screenshots shown.

**Comment by engram-design** [comment]
Yeah sorry about that - Looks like I moved on to other things without finishing it off! It's pretty close though, so be on the lookout. It essentially allows you to set what elements you wish to enable comments on.

**Comment by rdomingue** [comment]
In fact, I already did a commenting system because I wasn't able to make yours works. Well, I'm a front-end dev and with base notion of back-end. So I did it in SQL, php and Ajax as I didn't know how exactly works plugin of Craft based on Yii. So it's not as clean as yours but it works... and works inside the CP of craft too. 
So I'm good for now with my commenting system.

Thanks you, I will certainly take a look later on your plugin.
